walk/config.go

```diff
@@ -166,12 +167,75 @@ func loadBazelIgnore(repoRoot string) (isIgnoredFunc, error) {
 		excludes[ignore] = struct{}{}
 	}
 
-	isIgnored := func(p string) bool {
+	isBazelIgnored := func(p string) bool {
 		_, ok := excludes[p]
 		return ok
 	}
 
-	return isIgnored, nil
+	return isBazelIgnored, nil
+}
+
+func loadRepoDirectoryIgnore(repoRoot string) (isIgnoredFunc, error) {
+	ignorePath := path.Join(repoRoot, "REPO.bazel")
```

Comment:
Let's call this repoFilePath or similar

Reply:
Done. Thank you for pointing out my copy-paste errors 😅